compiler/src/dotty/tools/dotc/typer/ImportSuggestions.scala

```diff
@@ -300,7 +301,7 @@ trait ImportSuggestions:
       if filled < n && rest.nonEmpty then rest.toList.best(n - filled)
       else Nil
     top.take(filled).toList ++ remaining
-  end best
+  //end best  TODO: re-enable with new syntax
```

Comment:
What happen here?

Reply:
We want to replace this with `end extension` but that only works for bootstrapped